js/components/utils.js

```diff
@@ -481,10 +481,10 @@ var Utils = {
    * @function getPropertyValue
    * @param {Object} object - The object we want to extract the property form
    * @param {String} propertyPath - A path that points to a nested property in the object with a form like 'prop.nestedProp1.nestedProp2'
-   * @param {Object} defaltValue - (Optional) A default value to return when the property is undefined
+   * @param {Object} defaultValue - (Optional) A default value to return when the property is undefined
```

Comment:
Since @aeng7 had a comment, can we also add a note here in order to clarify that `null` will be returned instead of `defaultValue`?

Reply:
Will do.